api/controller/system.ts

```diff
@@ -0,0 +1,32 @@
+import Response from 'shared/model/response';
+import {BlockService} from 'api/service/block';
+
+const constants = require('../../backlog/helpers/constants');
+
+class RoundService {}
+
+@Controller('?')
```

Comment:
Is it correct?

Reply:
No it isn't, but the only thing I know about this handler is the action it should perform.